### DISCUSSION TITLE
Welcome to iWuBi Discussions!

### DISCUSSION THREAD

**Discussion by iHTCboy** [iHTCboy/iWuBi#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OnchiuLee** [comment]
建议加上更形象的字根拆分支持，如需显示资源，98五笔资源库可提供完美拆分字体与拆分码表支持  qq群547820540，
